src/lang-pt_BR.js

```diff
@@ -0,0 +1,517 @@
+(function (janela) {
+  /**
+   * {virtualclass1} {virtualclass2} are elements you passes with getString function eg:-
+   *   virtualclass.lang.getString('operaBrowserIssue', ['opeara', 27]);
+   *   opera and 27 will be replaced over the {virtualclass1} and {virtualclass2} resepectively for particular line of language file.
+   * @type type
+   */
+  const message = {
+    ActiveAll: 'Ativar todos',
+    addcontext: 'Adicionar marcador de páginas',
+    Addoption: 'Adicionar opção',
+    addnew: 'Adicionar novo',
+    adminusr: 'Usuário admin',

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482209)